tests_e2e/orchestrator/runbook.yml

```diff
@@ -8,12 +8,11 @@ extension:
   - "./lib"
 
 variable:
+  #
+  # These variables define runbook parameters; they are handled by LISA.
+  #
   - name: subscription_id
     value: ""
-  - name: keep_environment
-    value: "no"
-  - name: wait_delete
```

Comment:
Remove this runbook parameter. By default the runbook starts deleting the test VMs asynchronously and does not wait for the deletion to complete. This parameter can force a wait on the delete operation, but it is not needed by the automation pipeline.

I moved keep_environment to the end of the list of runbook parameters